Makefile

```diff
@@ -1,115 +1,98 @@
-.PHONY: uninstall install setup-cli get-master-ip launch-dcos detroy-dcos docker-build docker kubectl-tunnel deploy
+.PHONY: azure aws gce uninstall install get-cli setup-cli get-master-ip get-master-elb_ip plan-dcos launch-dcos detroy-dcos kubectl-config kubectl-tunnel plan deploy
 
 RM := rm -f
-LAUNCH_CONFIG_FILE := launch.yaml
-CLUSTER_INFO_FILE := cluster_info.json
-BUILD_FILE := .output
-MASTER_IP_FILE := .master_ip
-ID_FILE := .id_key
 SSH_USER := core
-DCOS_INSTALLER_URL := https://downloads.dcos.io/dcos/stable/1.10.1/dcos_generate_config.sh
-
-include ./resources/gce.template.mk
-include ./resources/aws.template.mk
-include ./resources/azure.template.mk
+MASTER_IP_FILE := .master_ip
+MASTER_ELB_IP_FILE := .master_elb_ip
+TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
+DCOS_LAUNCH_VERSION := 0.5.7
```

Comment:
Actually, you want the DC/OS CLI (`dcos`) version, not `dcos-launch` which you are deprecating in this PR.